backend/xivdpscalc/character/jobs.py

```diff
@@ -122,10 +134,25 @@ def create_job(name: str):
 
         return job_string_to_enum[name]
 
+@dataclass(init=False)
```

Comment:
Why are you using this? As-is, this decorator will generate repr and equality dunder methods — but I didn't see any usage of either throughout the rest of the code.

Reply:
pylint says it doesn't like classes with no public methods and after reading about dataclasses, this conceptually fits that definition of being a class just used to store data

Reply:
That's likely okay for now, but it's worth considering that `Comp` might end up with some properties or etc (for example, you could make `n_roles` a property). At that point, probably worth reviewing if removing the dataclass attribute.

I'd probably suggest putting a comment explaining why there's a dataclass here so that future decisions about whether to keep/remove it are informed.